.typos.toml

```diff
@@ -0,0 +1,33 @@
+[files]
+extend-exclude = [
+    ".git/",
+    "tests/Fixtures/Integration/misc/",
+    "tests/Fixtures/Integration/priority/",
+]
+ignore-hidden = false
+
+[default]
+extend-ignore-re = [
+    "Sebastiaan Stok",
+    "mettre sur",
+    "posa sur",
+    "\\S+…",
+    "\\$thi Hello!",
+    # Mixed case
+    "function __construcT",
+    "lIst\\{",
+    "const foO",
+    "\\$a->ba\\b",
+    "function foO",
+    "use BaRr",
+    "new foO",
+    "StatiC",
+    "__consTRUCT",
+    "__IsseT",
+    "\\bbaR",
+    "__clasS__",
+    "sTRING",
```

Comment:
I am wondering if it wouldn't be better to ignore whole `tests` directory, because there are a lot of nonsensical words and casing, not only in integration but also in unit tests. It can lead to CI failures and I don't see much value in maintaining this list in terms of tests.

Reply:
This is a great approach, I take care too, so I understand 🙂. But in terms of our tests, that sometimes contain typos on purpose I don't see much value in linting it and maintaining list of excludes... I think it's great to lint user-facing code, linting tests for typos may provide too much failures (again, there's no official way to run it locally). I'm just thinking out loud 🙂. @keradus what do you think?

Reply:
I do not like to have typos in `/tests/` folder.
as developer, it would hurt me to read code that is ugly. having SCA for _some_ part of code (eg `src/`) but not all (eg `tests/`) is not good. and tests are as much code as source files.

Reply:
@keradus but this would lead to maintaining `.typos.toml` everytime we do weird casing things in the tests 😕.

Reply:
let's have the tool installable locally and executable via composer scripts. then, i would be able to run the tool locally and guide on covering or excluding `tests/`

Reply:
@keradus since it's not a PHP tool it can't be executable via Composer scripts (as these should be compatible with Docker runtimes). It can be executable via Docker Compose service, like Sphinx and Markdown linters, though.